.template-helpers/variant.sh

```diff
@@ -23,9 +24,16 @@ for tag in "${tags[@]}"; do
 	fi
 done
 if [ "$text" ]; then
-	# give a little space
-	echo
-	echo
-	cat "$dir/variant.md"
-	echo "$text"
+  latest=($(curl -sSL $url | sed -e 's/git:\/\/github.com\///' -e 's/@/ /' - | grep "latest"))
+  dockerfile='https://raw.githubusercontent.com/'"${latest[1]}"'/'"${latest[2]}"'/'"${latest[3]}"'/Dockerfile'
+  base_image=$(curl -sSL $dockerfile | sed 's/:/\t/' | nawk '/^FROM .*$/ { print $2 }')
```

Comment:
Same comment on quoting (`curl -sSL "$dockerfile" ...`), and I'd also note that we stick to `camelCasing` for variable names in Bash scripts.  As a final note, that last `awk` could be simplified as `awk '$1 == "FROM" { print $2 }'`.